bluepyopt/electrical/parameters.py

```diff
@@ -93,6 +93,10 @@ def check_bounds(self):
                     (self.name, self._value, str(self.lower_bound),
                      str(self.upper_bound)))
 
+    def destroy(self):
```

Comment:
Wouldn't a consistent interface make it easier so one doesn't always have to check if a method exists? A no-op destroy reduces the differences between Parameter subclasses